pkg/pb/overlay.proto

```diff
@@ -6,34 +6,10 @@ option go_package = "pb";
 
 import "gogo.proto";
 import "node.proto";
-import "vouchers.proto";
 
 package overlay;
 
-service Nodes {
-    rpc Query(QueryRequest) returns (QueryResponse);
-    rpc Ping(PingRequest) returns (PingResponse);
-    rpc RequestInfo(InfoRequest) returns (InfoResponse);
-}
-
-message QueryRequest {
-    node.Node sender = 1;
-    node.Node target = 2;
-    int64 limit = 3;
-    bool pingback = 4;
-    repeated vouchers.Voucher vouchers = 5;
-}
-
-message QueryResponse {
-    node.Node sender = 1;
-    repeated node.Node response = 2;
-}
-
-message PingRequest {};
-message PingResponse {};
-
 // TODO: add fields that validate who is requesting the info
```

Comment:
this TODO can be removed as well. Also, is there a reason `InfoResponse` has not been removed below?

Reply:
oddly enough, InfoResponse is still used in quite a few places

Reply:
Hmm, well if that is the case, we should probably define a go struct for it rather than using a protobuf struct (assuming it is not used in rpc communication). But that doesn't need to happen in this PR.

Reply:
https://storjlabs.atlassian.net/browse/V3-2879